app/models/display_ad.rb

```diff
@@ -1,17 +1,25 @@
 class DisplayAd < ApplicationRecord
   resourcify
 
+  ALLOWED_PLACEMENT_AREAS = %w[sidebar_left sidebar_left_2 sidebar_right].freeze
```

Comment:
How are `sidebar_left` and `sidebar_left_2` different from each other? Could there be more descriptive names like e.g. `sidebar_left_top` and `sidebar_left_bottom`? If we change that we'd need a DUS to update the stored value, which is why I really prefer using Rails enums because they let you change names without changing this stored value in the DB but also give you the scopes, validations, and predicates for free.

Reply:
I went and added a human-readable value for display... which can change dynamically without us having to mess with the data.

<img width="440" alt="Screen Shot 2021-09-27 at 4 56 59 PM" src="https://user-images.githubusercontent.com/3102842/134984145-dc79197a-0718-42e3-aa7f-b747ac598084.png">

What if we accepted this as good enough for now and in the future did a separate data migration/DUS to get this value on to an enum as the longterm solution (vs overpowering this PR?)

Reply:
👍  on the good enough for now approach as far as I'm concerned.